### ISSUE TITLE
jsr292Test_jitCount0_SE80_0 crashes with Known Object Table disabled

### ISSUE THREAD

**Issue by dmitry-ten** [eclipse/openj9#3640]
Test segfaults, because `runFEMacro` case `java_lang_invoke_FilterArgumentsHandle_filterArguments` assumes that Known Object Table is enabled.
The easiest thing to do would be to check if KOT is enabled before accessing it, but it seems that the code relies heavily on the table existing, so I'm not sure if simply skipping it wouldn't affect correctness.

**Comment by dmitry-ten** [comment]
@mpirvu this is relevant to JITaaS, since we do not support Known Object Table.

**Comment by mpirvu** [comment]
Does the non-jitaas build run with disableKnownObjectTable? I assume not because we would crash when accessing it here:
```
            TR::KnownObjectTable *knot = comp()->getOrCreateKnownObjectTable();
            for (int i = 0; i <numFilters; i++)
               {
               filterIndexList[i] = knot->getIndex(fej9->getReferenceElement(filters, i));
               }
```
It's worth asking someone from the opt team what is the right/safe path for this code when KOT is disabled and then follow that path for JITaaS.

**Comment by dmitry-ten** [comment]
No, it doesn't run with disabled known object table.
But I think it needs to be changed in baseline code as well, because just assuming that it's enabled doesn't seem right.

**Comment by mpirvu** [comment]
Correct, the baseline needs to be changed to allow for disableKnownObjectTable option
